source/Engine.cpp

```diff
@@ -2050,7 +2050,8 @@ void Engine::HandleMouseClicks()
 						else
 						{
 							activeCommands |= Command::LAND;
-							Messages::Add("Landing on " + planet->Name() + ".", Messages::Importance::High);
+							if(!flagship->IsDestroyed())
```

Comment:
You can move this to the `else` above. It doesn't make sense to add the command if you don't send the message.